.github/workflows/functional.yml

```diff
@@ -86,8 +80,13 @@ jobs:
           SECRETS_RSTUF_ADMIN_PASSWORD: test-secret
 
     steps:
-      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
-      - uses: actions/setup-python@c4e89fac7e8767b327bbad6cb4d859eda999cf08
+      - name: Checkout release tag
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+
+      - name: Set up Python
+        uses: actions/setup-python@c4e89fac7e8767b327bbad6cb4d859eda999cf08
```

Comment:
Nit: can you leave a comment about the actual version pointed out by the hash (as suggested above)?

Reply:
I find most of these will be covered by dependabot either way so we can skip it 👍